src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsEnums.cs

```diff
@@ -938,7 +938,8 @@ internal enum FedAuthInfoId : byte
 
         // Data Classification constants
         internal const byte DATA_CLASSIFICATION_NOT_ENABLED = 0x00;
-        internal const byte MAX_SUPPORTED_DATA_CLASSIFICATION_VERSION = 0x01;
+        internal const byte DATA_CLASSIFICATION_VERSION_WITHOUT_RANK_SUPPORT = 0x01;
+        internal const byte MAX_SUPPORTED_DATA_CLASSIFICATION_VERSION = 0x02;
```

Comment:
If this isn't a translation of a macro from the Sql native headers it might be useful to rename it to something like 
`DATA_CLASSIFICATION_VERSION_MAX_SUPPORTED` so that the words data classification always come first.

Reply:
It isn't new, but I think it's no norm either. If you see underneath in next lines.. there's MAX/MIN TCE related enums. Probably driving factor for this one.